src/config/multiple-creds-same-aid-config.json

```diff
@@ -0,0 +1,117 @@
+{
+  "secrets": {
+    "gleif1": "D_PbQb01zuzQgK-kDWjq5",
+    "qvi1": "BTaqgh1eeOjXO5iQJp6m5",
+    "le1": "Akv4TFoiYeHNqzj3N8gE5",
+    "ecr1": "nf98hUHUy8Vt5tvdyaYV5"
+  },
+  "credentials": {
+    "gleif_to_qvi_vlei_cred": {
+      "type": "direct",
+      "schema": "QVI_SCHEMA_SAID",
+      "privacy": false,
+      "attributes": {}
+    },
+    "qvi_to_le_vlei_cred": {
+      "credSource": {
+        "type": "qvi"
+      },
+      "type": "direct",
+      "schema": "LE_SCHEMA_SAID",
+      "rules": "LE_RULES",
+      "privacy": false,
+      "attributes": {}
+    },
+    "le_to_ecr_birth_doula": {
+      "credSource": {
+        "type": "le"
+      },
+      "type": "direct",
+      "schema": "ECR_SCHEMA_SAID",
+      "rules": "ECR_RULES",
+      "privacy": true,
+      "attributes": {
+        "engagementContextRole": "Birth Certified Doula"
```

Comment:
for vlei-verifier-workflows lets use generic role names like "employee role 1", etc.